debian/bionic/foreman/foreman-service.install

```diff
@@ -0,0 +1 @@
+extras/systemd/foreman.service /usr/lib/systemd/system
```

Comment:
turns out, this needs to be `/lib`, not `/usr/lib` for some magic scripts in debian to work :woman_facepalming: